buildtest/buildsystem/base.py

```diff
@@ -393,7 +397,7 @@ class data structures that are used by both run and dry_run
                 self.metadata[known_section] = self.recipe.get(known_section)
 
         # Get the shell (sh, bash) for writing testscript. A Buildspec could specify this via ``shell`` key
-        self.metadata["shell"] = self.get_shell()
+        self.metadata["shell"] = self.shell
```

Comment:
Why is this added to metadata?

Reply:
Ill remove it for now, it was not clear how this metadata will be used going forward one thought was to use the metadata to write to a json file for reporting (``buildtest build report``) #284